changes/4642.documentation

```diff
@@ -0,0 +1 @@
+Added `documentation` category to release-notes.
```

Comment:
Should this be a housekeeping changelog?

Reply:
Defining categories is hard! :-( I figure since it's a change to the structure of the existing release-notes it makes sense to report as a documentation change.